### PR TITLE
fix java documentation snippet by being explicit

### PR DESCRIPTION
**Pull request by ktoso**
This looks good in docs.

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

File: akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java [hunk 1]
```diff
@@ -57,11 +53,13 @@
...
-      new OneForOneStrategy(10, Duration.create(1, TimeUnit.MINUTES), DeciderBuilder.
-        match(ArithmeticException.class, e -> resume()).
-        match(NullPointerException.class, e -> restart()).
-        match(IllegalArgumentException.class, e -> stop()).
-        matchAny(o -> escalate()).build());
+      new OneForOneStrategy(10, Duration.create(1, TimeUnit.MINUTES),
+          DeciderBuilder
+              .match(ArithmeticException.class, e -> SupervisorStrategy.resume())
+              .match(NullPointerException.class, e -> SupervisorStrategy.restart())
+              .match(IllegalArgumentException.class, e -> SupervisorStrategy.stop())
```
Comment (johanandren) [inline akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java]: Isn't the `SupervisorStrategy` missing from the imports now?

Comment (ktoso) [reply akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java]: Goddamn ;) Yes, thanks

File: akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java [hunk 2]
```diff
@@ -29,10 +29,7 @@
 
 //#supervisor
 import akka.japi.pf.DeciderBuilder;
-import static akka.actor.SupervisorStrategy.resume;
-import static akka.actor.SupervisorStrategy.restart;
-import static akka.actor.SupervisorStrategy.stop;
-import static akka.actor.SupervisorStrategy.escalate;
+import static akka.actor.SupervisorStrategy;
```
Comment (johanandren) [inline akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java]: It's just a class so it doesn't need to be static? ;)

File: akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java [hunk 3]
```diff
@@ -89,10 +88,11 @@ public Receive createReceive() {
...
       new OneForOneStrategy(10, Duration.create(1, TimeUnit.MINUTES), DeciderBuilder.
-        match(ArithmeticException.class, e -> resume()).
-        match(NullPointerException.class, e -> restart()).
-        match(IllegalArgumentException.class, e -> stop()).
-        matchAny(o -> escalate()).build());
+        match(ArithmeticException.class, e -> SupervisorStrategy.resume()).
+        match(NullPointerException.class, e -> SupervisorStrategy.restart()).
+        match(IllegalArgumentException.class, e -> SupervisorStrategy.stop()).
+        matchAny(o -> SupervisorStrategy.escalate())
+        .build());
```
Comment (ktoso) [inline akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java]: here too..

**Review by patriknw — COMMENTED** [review]
good

**Pull request merged**